app/views/home/index.html.erb

```diff
@@ -1,12 +1,8 @@
 <%= react_component("CyclingBackground", {bgPath:get_random_bg_image}) %>
 <% draffle = Draffle.all.first %>
-<% if !draffle.nil? %>
-  <%= link_to "/draffles/#{draffle.id}", class: "button draffle-view-button" do %>
-    View<br /><%= draffle.name %>
-  <% end %>
-<% end %>
 <%= react_component("Rankings", {
   date: @match_date,
   rankedPlayers: @ranked_players,
-  unrankedPlayers: @unranked_players
+  unrankedPlayers: @unranked_players,
+  draffle: draffle.id
```

Comment:
This breaks if there is no active draffle

Reply:
Ahh, an optional chain will fix that yeah?  draffle?.id or is there something more finicky there?

Reply:
Do I need to be more explicit
`!draffle.nil ? draffle.id : false  `